### DISCUSSION TITLE
HWO and Short Term Forecast

### DISCUSSION THREAD

**Discussion by branmh** [weather-gov/api#265] [Q&A] [answered]
Where can I find Hazardous Weather Outlook and Short Term Forecast via the API?

**Comment by JJKraw** [comment, marked as answer]
Hazardous weather outlook:

  https://api.weather.gov/products/types/HWO/locations/ {office code}

Short Term Forecast:

  https://api.weather.gov/products/types/NOW/locations/ {office code}

You can get the list of products at:

  https://api.weather.gov/products/types
